tests/packagedcode/data/cargo/cargo_lock/sample2/Cargo.lock

```diff
@@ -0,0 +1,13 @@
+# This file is automatically @generated by Cargo.
+# It is not intended for manual editing.
+[[package]]
+name = "autocfg"
+version = "1.0.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "f8aac770f1885fd7e387acedd76065302551364496e46b3dd00860b2f8359b9d"
```

Comment:
I wonder if we are not missing out on something by skipping the `source` and `checksum`. Can you enter a follow up ticket to revisit this later? We might want to add some new fields to the `DependentPackage` for that as these look on the surface as being quite similar to what we have in a go.mod and go.sum? Or we may want to keep these as qualifiers? 
And thinking of it... there are also cases in Python and Ruby and npm where we get deps fetched from git repos